libraries/AP_RangeFinder/AP_RangeFinder.cpp

```diff
@@ -175,8 +175,8 @@ RangeFinder::RangeFinder()
 }
 
 void RangeFinder::convert_params(void) {
-    if (params[0].type.configured_in_storage()) {
-        // _params[0]._type will always be configured in storage after conversion is done the first time
+    if (params[0].type.configured()) {
+        // _params[0]._type will always be configured after conversion is done the first time
```

Comment:
This comment is somewhat misleading.  No conversion will be done if it is `configured()` because it's in a defaults file.

Reply:
i've removed that one and I think @hendjoshsr71 will fix it separately

Reply:
i've expanded the comment, but explaining configured() in each place will quickly become tedious